flow/ProtocolVersions.cmake

```diff
@@ -81,12 +81,12 @@ set(FDB_PV_NETWORK_ADDRESS_HOSTNAME_FLAG        "0x0FDB00B071010000LL")
 set(FDB_PV_STORAGE_METADATA                     "0x0FDB00B071010000LL")
 set(FDB_PV_PERPETUAL_WIGGLE_METADATA            "0x0FDB00B071010000LL")
 set(FDB_PV_STORAGE_INTERFACE_READINESS          "0x0FDB00B071010000LL")
-set(FDB_PV_TENANTS                              "0x0FDB00B071010000LL")
 set(FDB_PV_RESOLVER_PRIVATE_MUTATIONS           "0x0FDB00B071010000LL")
-set(FDB_PV_OTEL_SPAN_CONTEXT                    "0x0FDB00B072000000LL")
-set(FDB_PV_SW_VERSION_TRACKING                  "0x0FDB00B072000000LL")
-set(FDB_PV_ENCRYPTION_AT_REST                   "0x0FDB00B072000000LL")
-set(FDB_PV_SHARD_ENCODE_LOCATION_METADATA       "0x0FDB00B072000000LL")
-set(FDB_PV_BLOB_GRANULE_FILE                    "0x0FDB00B072000000LL")
-set(FDB_ENCRYPTED_SNAPSHOT_BACKUP_FILE          "0x0FDB00B072000000LL")
-set(FDB_PV_CLUSTER_ID_SPECIAL_KEY               "0x0FDB00B072000000LL")
+set(FDB_PV_OTEL_SPAN_CONTEXT                    "0x0FDB00C071200000LL")
```

Comment:
All of these feature versions that existed on snowflake/release-71.2 need to be the same as there. (https://github.com/apple/foundationdb/blob/snowflake/release-71.2/flow/ProtocolVersions.cmake)
There are several sources of these differences
- that apple's 7.1 and snowflake's 7.1 had different branch points, and thus some features between those are different.
- some of the features in apple's 7.2 were already in snowflake's 71.2, but some of them are in this release (71.3)

So specifically, these do not match release-71.2's protocol versions:
FDB_PV_RESOLVER_PRIVATE_MUTATIONS
FDB_PV_TENANTS

Reply:
Done